webapp/tests/forms/steps/lotse/test_personal_data_steps.py

```diff
@@ -491,3 +491,33 @@ def test_if_number_over_25_chars_then_succ_validation(self, new_test_request_con
             step = LotseStepChooser().get_correct_step(StepTelephoneNumber.name, True, ImmutableMultiDict(data))
             form = step.render_info.form
             assert form.validate() is False
+
+class TestPersonAHasDisabilityValidation:
+    def test_required_value_is_give_validation_should_be_success(self, new_test_request_context):
```

Comment:
Dou you mean
```suggestion
    def test_if_required_value_is_given_then_validation_should_be_success(self, new_test_request_context):
```
? Same for the other tests. :)